uint16 id
geometry_msgs/Pose goal
bool cartesian
bool keep_position
bool keep_orientation